src/suricata.c

```diff
@@ -2698,6 +2692,12 @@ int main(int argc, char **argv)
         FlowDisableFlowRecyclerThread();
     }
 
+    if (suri.run_mode != RUNMODE_UNIX_SOCKET) {
+        /* we need a packet pool for FlowForceReassembly */
+        PacketPoolInit();
+        FlowForceReassembly();
```

Comment:
Whats the logic of moving this to after the packet processing threads are killed? How are timeout pseudo packets processed?